### DISCUSSION TITLE
.ang file does not open in MTEX 5.10.0, but opens in 5.8.1

### DISCUSSION THREAD

**Discussion by Savillian** [mtex-toolbox/mtex#1905] [Ask Anything] [answered]
Hi everyone,

I was trying out the latest version of MTEX when I discovered version 5.10.0 reads .ang files differently. I'm not sure how, but a .ang file that I can open with version 5.8.1 is incompatible with the latest version.

MTEX appears to recognize the file (the green loading box appears on starting the import command) but something at the end of the loading process leads to failure.

Error follows:
**Error using loadEBSD_ang
EBSD format 'ANG' does not match the data

Error in EBSD.load (line 86)
  ebsd{k} = feval(['loadEBSD_',char(interface)],...

Error in ReconstructionReferenceScript (line 71)
ebsd = EBSD.load(fname,CS,'interface','ang',**

So far I haven't found documentation explaining this issue - does anyone have insight on this? 

Thanks in advance,
Alec

**Comment by kilir** [comment]
Hi,
any chance you could provide a small example file?
Cheers,
Rüdiger

**Comment by Savillian** [comment]
Hi Rüdiger,

The only file I've had the issue with is 180 MB in size - happy to send it
your way if there's an easy method. So far none of my smaller scans have
this issue (unfortunately).

I've been testing all the EBSD .ang files I can find, but so far only one
shows this issue. I also couldn't find anything wrong in the text of the
.ang file when compared to files that do work in both versions - hence my
forum post.

All the best,
Alec

On Tue, Jul 25, 2023 at 4:31 AM Rüdiger Kilian ***@***.***>
wrote:

> Hi,
> any chance you could provide a small example file?
> Cheers,
> Rüdiger
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mtex-toolbox/mtex/discussions/1905#discussioncomment-6538882>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJCJ6SZQL4T3LMTBD6LHVY3XR6OBTANCNFSM6AAAAAA2TMW4UE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Alec Saville

No trees were killed in the process of sending this message; however a
large number of electrons were terribly inconvenienced...

**Reply by ralfHielscher** [reply]
Maybe any file-sharing platform like dropbox? You should also be able to upload the data on github.
Ralf.

**Comment by Savillian** [comment]
I uploaded the dataset to Dropbox. 


On Sun, Aug 6, 2023 at 2:24 AM Ralf Hielscher ***@***.***>
wrote:

> Maybe any file-sharing platform like dropbox? You should also be able to
> upload the data on github.
> Ralf.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mtex-toolbox/mtex/discussions/1905#discussioncomment-6649193>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJCJ6S5YH3ZQST2T4TO5CPTXT5IDFANCNFSM6AAAAAA2TMW4UE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Alec Saville

No trees were killed in the process of sending this message; however a
large number of electrons were terribly inconvenienced...

**Reply by kilir** [reply]
Hi,

loading works for me, latest mtex version, either like below 
```
ebsd = loadEBSD_ang('1050HIPtemper_1_N_XZ.ang')
 
ebsd = EBSD
 
 Phase    Orientations           Mineral         Color  Symmetry  Crystal reference frame
     1  1548814 (100%)  Titanium (Alpha)  LightSkyBlue       622       X||a, Y||b*, Z||c*
     2    566 (0.037%)   Titanium (Beta)  DarkSeaGreen       432                         
 
 Properties: ci, fit, iq, sem_signal, unknown1, unknown2, unknown3, unknown4, x, y, oldId
 Scan unit : um
```
or through the script generated by the import_wizard

```
% crystal symmetry
CS = {... 
  'notIndexed',...
  crystalSymmetry('622', [3 3 4.7], 'X||a', 'Y||b*', 'Z||c*', 'mineral', 'Titanium (Alpha)', 'color', [0.53 0.81 0.98]),...
  crystalSymmetry('432', [3.3 3.3 3.3], 'mineral', 'Titanium (Beta)', 'color', [0.56 0.74 0.56])};

ebsd = EBSD.load(fname,CS,'interface','ang', 'convertEuler2SpatialReferenceFrame');
```
Cheers,
Rüdiger

**Comment by Savillian** [comment, marked as answer]
I'm seeing the same thing now - I have no idea what changed over the last
few weeks, but I'll close the ticket on GitHub.

I'm guessing it was a Windows update thing - had several updates the last
few weeks.

On Tue, Aug 8, 2023 at 1:29 AM Rüdiger Kilian ***@***.***>
wrote:

> Hi,
>
> loading works for me, latest mtex version, either like below
>
> ebsd = loadEBSD_ang('1050HIPtemper_1_N_XZ.ang')
>
> ebsd = EBSD
>
>  Phase    Orientations           Mineral         Color  Symmetry  Crystal reference frame
>      1  1548814 (100%)  Titanium (Alpha)  LightSkyBlue       622       X||a, Y||b*, Z||c*
>      2    566 (0.037%)   Titanium (Beta)  DarkSeaGreen       432
>
>  Properties: ci, fit, iq, sem_signal, unknown1, unknown2, unknown3, unknown4, x, y, oldId
>  Scan unit : um
>
> or through the script generated by the import_wizard
>
> % crystal symmetry
> CS = {...
>   'notIndexed',...
>   crystalSymmetry('622', [3 3 4.7], 'X||a', 'Y||b*', 'Z||c*', 'mineral', 'Titanium (Alpha)', 'color', [0.53 0.81 0.98]),...
>   crystalSymmetry('432', [3.3 3.3 3.3], 'mineral', 'Titanium (Beta)', 'color', [0.56 0.74 0.56])};
>
> ebsd = EBSD.load(fname,CS,'interface','ang', 'convertEuler2SpatialReferenceFrame');
>
> Cheers,
> Rüdiger
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mtex-toolbox/mtex/discussions/1905#discussioncomment-6666152>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJCJ6SZLAXARBTGON5QWIQ3XUHTDZANCNFSM6AAAAAA2TMW4UE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Alec Saville

No trees were killed in the process of sending this message; however a
large number of electrons were terribly inconvenienced...
